types/prosemirror-view/index.d.ts

```diff
@@ -323,7 +323,7 @@ export class EditorView<S extends Schema = any> {
    * and `right` will be the same number, as this returns a flat
    * cursor-ish rectangle.
    */
```

Comment:
^ Can you please copy over the new comment from https://github.com/ProseMirror/prosemirror-view/blob/13b046a834b489530a98dd362fa55703e52e076d/src/index.js#L279-L284 ?

Reply:
Good catch. Done!